### PR TITLE
Create CHANGELOG.md

### PR DESCRIPTION
**Pull request by balavishnuvj**

### CODE REVIEW

**Comment by msutkowski** [comment]
Hey @balavishnuvj, the `CHANGELOG` you're looking for can be found here: https://github.com/mswjs/msw/releases. An example of a `CHANGELOG.md` file that would better suit this repo can be seen here if you'd like to mimic it and update this PR: https://github.com/reduxjs/redux/blob/master/CHANGELOG.md.

Thanks!

**Comment by balavishnuvj** [comment]
@msutkowski it was very similar, I had adapted from https://github.com/testing-library/react-testing-library/blob/master/CHANGELOG.md

Have updated though :)

**Comment by marcosvega91** [comment]
@balavishnuvj unfortunately MSW is not configured to work with  [semantic-release](https://github.com/semantic-release/semantic-release). `semantic-release` is a tool to auto release versions based on commits. For now this process is manually.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# CHANGELOG
```
Comment (kettanaito) [inline CHANGELOG.md]: Nitpick: Let's write this in a regular case: `# Change Log`.

**Review by kettanaito — CHANGES_REQUESTED** [review]
One nitpick and it's good to go!

**Review by kettanaito — APPROVED** [review]

**Pull request merged**